pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -404,6 +405,12 @@ func (ctx *context) getDIDDocAndConnection(pubDID string) (*did.Doc, *Connection
 			return nil, nil, fmt.Errorf("resolve public did[%s]: %w", pubDID, err)
 		}
 
+		// TODO: x.y.z.foo
```

Comment:
?

Reply:
I'll remove the TODO - this is part of issue #1004